src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionXpathFilterTest.java

```diff
@@ -179,4 +179,108 @@ private static SuppressionXpathFilter createSuppressionXpathFilter(String fileNa
+        final TreeWalkerAuditEvent evQuot = new TreeWalkerAuditEvent(null,
+                "Test.java", messageQuot, JavaParser.parseFile(file,
+                JavaParser.Options.WITHOUT_COMMENTS));
+        assertFalse(filter.accept(evQuot), "TreeWalker audit event should be rejected");
+
+        final LocalizedMessage messageLess = new LocalizedMessage(8, 33, TokenTypes.PLUS, "", "",
+                null, null, "777", getClass(), null);
+        final TreeWalkerAuditEvent evLess = new TreeWalkerAuditEvent(null,
+                "Test.java", messageLess, JavaParser.parseFile(file,
+                JavaParser.Options.WITHOUT_COMMENTS));
+        assertFalse(filter.accept(evLess), "TreeWalker audit event should be rejected");
+
+        final LocalizedMessage messageAmpersand = new LocalizedMessage(11, 38, TokenTypes.PLUS,
+                "", "", null, null, "777", getClass(), null);
+        final TreeWalkerAuditEvent evAmpersand = new TreeWalkerAuditEvent(null,
+                "Test.java", messageAmpersand, JavaParser.parseFile(file,
+                JavaParser.Options.WITHOUT_COMMENTS));
+        assertFalse(filter.accept(evAmpersand), "TreeWalker audit event should be rejected");
+
+        final LocalizedMessage messageGreater = new LocalizedMessage(14, 36, TokenTypes.PLUS,
+                "", "", null, null, "777", getClass(), null);
+        final TreeWalkerAuditEvent evGreater = new TreeWalkerAuditEvent(null,
+                "Test.java", messageGreater, JavaParser.parseFile(file,
+                JavaParser.Options.WITHOUT_COMMENTS));
+        assertFalse(filter.accept(evGreater), "TreeWalker audit event should be rejected");
+
+        final LocalizedMessage messageNewLine = new LocalizedMessage(17, 37, TokenTypes.PLUS,
+                "", "", null, null, "777", getClass(), null);
+        final TreeWalkerAuditEvent evNewLine = new TreeWalkerAuditEvent(null,
+                "Test.java", messageNewLine, JavaParser.parseFile(file,
+                JavaParser.Options.WITHOUT_COMMENTS));
+        assertFalse(filter.accept(evNewLine), "TreeWalker audit event should be rejected");
+
+        final LocalizedMessage messageSpecial = new LocalizedMessage(20, 37, TokenTypes.PLUS,
+                "", "", null, null, "777", getClass(), null);
+        final TreeWalkerAuditEvent evSpecial = new TreeWalkerAuditEvent(null,
+                "Test.java", messageSpecial, JavaParser.parseFile(file,
+                JavaParser.Options.WITHOUT_COMMENTS));
+        assertFalse(filter.accept(evSpecial), "TreeWalker audit event should be rejected");
+
+        final LocalizedMessage messageApos = new LocalizedMessage(23, 34, TokenTypes.PLUS,
+                "", "", null, null, "777", getClass(), null);
+        final TreeWalkerAuditEvent evApos = new TreeWalkerAuditEvent(null,
+                "Test.java", messageApos, JavaParser.parseFile(file,
+                JavaParser.Options.WITHOUT_COMMENTS));
```

Comment:
```
final LocalizedMessage messageApos = new LocalizedMessage(23, 34, TokenTypes.PLUS,
                "", "", null, null, "777", getClass(), null);
        final TreeWalkerAuditEvent evApos = new TreeWalkerAuditEvent(null,
                "Test.java", messageApos, JavaParser.parseFile(file,
                JavaParser.Options.WITHOUT_COMMENTS));
```
All these lines duplicated many times, only line number and column number are different. Please extract it to method that creates TreeWalkerAuditEvent

Reply:
All right. On it.

Reply:
Done